test-runner/src/main/scala/com/typesafe/spark/test/mesos/ClientModeSpec.scala

```diff
@@ -5,8 +5,6 @@ import org.scalatest.concurrent.TimeLimitedTests
 
 import com.typesafe.spark.test.mesos.framework.runners.RoleConfigInfo
 
-import MesosIntTestHelper.TEST_TIMEOUT
```

Comment:
Better place all imports at the top of the file.
The reader can refer to all imports in one place. So we should remove the local import:
import MesosIntTestHelper._
eg.